lib/FrontendTool/ReferenceDependencies.cpp

```diff
@@ -216,8 +227,10 @@ void ReferenceDependenciesEmitter::emit() const {
 
 bool swift::emitReferenceDependencies(DiagnosticEngine &diags, SourceFile *SF,
                                       const DependencyTracker &depTracker,
+                                      const ReferencedNameTracker *nameTracker,
```

Comment:
You may not even have to pass in a boolean, if you can get to the langopts from the ASTContext from the SourceFIle.